### ISSUE TITLE
compatibility with BMP180

### ISSUE THREAD

**Issue by JonathancalderonIL** [cano64/ATTiny85-ATTiny84-BMP085-Arduino-Library-FastAltitude#3]
hello,
will this work for the newer bmp180 >?
i am looking to have a BMP180 light a different led color for different ranges of altitudes.
any chance you can help ? 
this will be implumented on my FPV night flight hobby

**Comment by cano64** [comment]
Yes, it will work with no changes required at all, I have tested it and I
already build a device with BMP180.

**Comment by JonathancalderonIL** [comment]
thanks for the quick reply 
i am very new to arduino... 
can you give me a simple example code to read the atlitude ?
and how can i use this reading to light a led ?

**Comment by cano64** [comment]
it comes with example sketch in the example folder. To read altitude in
decimeters just use
int altSTDdm = bmp.readAltitudeSTDdm();

**Comment by JonathancalderonIL** [comment]
thanks 
is there a way reading altitude in meters? 
and i cant find were to connect the SDA and SCL to the ATTINY85 ...

**Comment by cano64** [comment]
SDA and SCL on tiny85 are on pins 5 and 7 respectively, see this diagram
http://fc04.deviantart.net/fs70/f/2013/038/3/7/attiny_web_by_pighixxx-d5u4aur.png
To get altitude in meters just divide the result in dm by 10.
There is also a method that returns altitude in meters as a float, but you
should avoid using floats on a microcontroller.

**Comment by beicnet** [comment]
Dear @cano64,

 I read that it's compatible with BMP180, so, it would work with BMP280 too?

Thank you!

Regards
